.github/workflows/genesis-verify.yml

```diff
@@ -0,0 +1,59 @@
+name: genesis-verify
```

Comment:
should this be a sub-task of gnoland.yml?

Reply:
It should definitely be separate, it has nothing to do with the `gnoland` binary, but is a very specific workflow for testnet deployments

Reply:
this file "Running latest gnoland with each genesis file" other than just `verify` so it's not totally `separated`.

the question is what is the situation a genesis file got verified but can not run successfully with gnoland binary?

Reply:
>the question is what is the situation a genesis file got verified but can not run successfully with gnoland binary?

Oof, for up to date genesis files, there would need to be an implementation mismatch between `gnogenesis` and `gnoland` (which is possible, not entirely out of the realm of reality).

There can be a situation like we do now, where the `test4` genesis was valid at the time, but it would fail verification now because it's an outdated genesis format (doesn't support genesis transactions)